props/saber_fett263_buttons.h

```diff
@@ -293,248 +660,3355 @@ class SaberFett263Buttons : public virtual PropBase {
+            MenuExit();
+            break;
+        }
+        break;
+  #if NUM_BLADES > 1          
+      case MENU_BLADE_STYLE:
+        if (blade_num_ == 0) {
+          PlayMenuSound("mblade.wav");
+          break;
+        }
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        menu_type_ = MENU_STYLE;
+        num_presets_ = current_config->num_presets;
+        tmp = current_preset_.GetStyle(blade_num_);
+        style_num_ = FirstWord(tmp, "builtin") ? atoi(SkipWord(tmp)) : 0; 
+        style_revert_ = style_num_;               
+        MenuSelect();
+        break;
+      case MENU_BLADE_COLOR:
+  #if NUM_BLADES > 2
+        menu_type_ = MENU_COLOR_SUB;
+        menu_sub_pos_ = 0;
+        copy_blade_ = blade_num_;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        PlayMenuSound("mcolorsb.wav");
+        break;
+  #else
+        menu_type_ = MENU_COLOR_SUB;
+        if (blade_num_ == 1) {
+          copy_blade_ = 2;
+        } else {
+          copy_blade_ = 1;
+        }
+        menu_sub_pos_ = 0;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        PlayMenuSound("moption.wav");
+        break;
+  #endif
+      case MENU_BLADE_COPY:
+        menu_type_ = MENU_COPY_COLOR;
+        SetColorCopy();
+        MenuConfirm();
+        break;
+      case MENU_BLADE_SETTING:
+        if (blade_num_ == 0) {
+          PlayMenuSound("mblade.wav");
+          break;
+        }
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        menu_type_ = MENU_STYLE_SETTING_SUB;
+        PlayMenuSound("moption.wav");
+        break;
+      case MENU_BLADE_LENGTH:
+        PlayMenuSound("mselect.wav");
+        show_preview_.Stop(blade_preview_);
+        SaveState(current_preset_.preset_num);
+        blade_preview_ = 0;
+        max_length_ = GetMaxBladeLength(blade_num_);
+        blade_length_ = GetBladeLength(blade_num_);
+        if (blade_length_ < 1) blade_length_ = max_length_;
+        length_revert_ = blade_length_;
+        LengthPreview(blade_num_);
+        menu_type_ = MENU_LENGTH;
+        break;
+      case MENU_COPY_COLOR:
+        if (choice_) {
+          current_preset_.SetStyle(copy_blade_, style_parser.CopyArguments(current_preset_.GetStyle(blade_num_), current_preset_.GetStyle(copy_blade_)));
+          current_preset_.Save();
+          show_preview_.Stop(blade_preview_);
+          UpdateStyle(current_preset_.preset_num);
+          menu_type_ = MENU_COLOR_SUB;
+          MenuSelect();
+          choice_ = false;
+          blade_preview_ = 0;
+          break;
+        }          
+        PlayMenuSound("mconfirm.wav");
+        break;
+  #endif           
+      case MENU_STYLE:
+        restore_point = nullptr; 
+        current_preset_.Save();
+  #if NUM_BLADES == 1
+        menu_type_ = MENU_TOP;
+        MenuSave();
+  #else           
+        menu_type_ = MENU_BLADE_STYLE;
+        MenuSave();
```

Comment:
move MenuSave() after the #endif since it's the same in both cases.

Reply:
Updated